src/App/store/Store.js

```diff
@@ -132,6 +132,25 @@ class Store {
   @action.bound
   getBackground() {
     if (localStorage.background) {
+      var background = JSON.parse(localStorage.background);
```

Comment:
nit: use `const` here (or `let` in other places) rather than `var` to make the intent clearer. Here and below.